Source/Core/Core/HW/DVD/DVDInterface.h

```diff
@@ -97,12 +124,23 @@ bool UpdateRunningGameMetadata(std::optional<u64> title_id = {});
 
 // Direct access to DI for IOS HLE (simpler to implement than how real IOS accesses DI,
 // and lets us skip encrypting/decrypting in some cases)
-void ChangePartition(const DiscIO::Partition& partition);
-void ExecuteCommand(u32 command_0, u32 command_1, u32 command_2, u32 output_address,
-                    u32 output_length, bool reply_to_ios);
+void ExecuteCommand(ReplyType reply_type);
+void PerformDecryptingRead(u32 position, u32 length, u32 output_address,
+                           const DiscIO::Partition& partition, ReplyType reply_type);
+// Exposed for use by emulated BS2; does not perform any checks on drive state
+void AudioBufferConfig(bool enable_dtk, u8 dtk_buffer_length);
+
+void SetLowError(u32 low_error);
+void SetHighError(u32 high_error);
 
 // Used by DVDThread
 void FinishExecutingCommand(ReplyType reply_type, DIInterruptType interrupt_type, s64 cycles_late,
                             const std::vector<u8>& data = std::vector<u8>());
 
+// Used by IOS HLE
+void SetInterruptEnabled(DIInterruptType interrupt, bool enabled);
+void ClearInterrupt(DIInterruptType interrupt);
+
+// Used by EXI_DeviceIPL
+extern u32 s_rtc_flag;
```

Comment:
I've tweaked this to also use the `Flags` template, and moved it into `EXI_DeviceIPL` since that's (presumably) where it's physically located and IMO it makes a bit more sense.  Let me know how that looks.